src/formatting_stack/linters/kondo.clj

```diff
@@ -11,22 +11,15 @@
 
 (def default-options
   {:cache     true
-   :cache-dir formatting-stack.kondo-classpath-cache/cache-dir
```

Comment:
Please extract the :cache-dir changes to a separate PR, preferably to an issue first so it can be discussed carefully.

As a brief experience report, java `nio` locking is not guaranteed to work (as reflected in its own javadocs), which by coincidence I happen to have verified with the **exact** same code that clj-kondo uses.

I suspect it has to do with OS/filesystem/encryption combination.

Please as a measure of caution don't introduce changes known to be broken on certain OSes.